beacon-chain/rpc/eth/beacon/structs.go

```diff
@@ -346,6 +346,16 @@ type BlsToExecutionChange struct {
 	ToExecutionAddress string `json:"to_execution_address" validate:"required"`
 }
 
+type ValidatorCountResponse struct {
+	ExecutionOptimistic string          `json:"execution_optimistic"`
+	Finalized           string          `json:"finalized"`
+	Data                *ValidatorCount `json:"data"`
+}
+
+type ValidatorCount struct {
+	ValidatorCount string `json:"validator_count"`
+}
```

Comment:
I think we can make this endpoint more useful if we support querying for multiple statuses.
```
type ValidatorCountResponse struct {
	ExecutionOptimistic string          `json:"execution_optimistic"`
	Finalized           string          `json:"finalized"`
	Data                *[]ValidatorCount `json:"data"`
}

type ValidatorCount struct {
    Status string `json:"status"`
	Count string `json:"count"`
}
```
That way users can write something like `&status=active,pending`. We could potentially return all statuses if the query is empty.

Reply:
Added support for multiple statuses